src/Files.Uwp/Helpers/FileExtensionHelpers.cs

```diff
@@ -47,14 +48,15 @@ public static bool IsPowerShellFile(string fileExtensionToCheck)
         /// otherwise <c>false</c>.</returns>
         public static bool IsZipFile(string fileExtensionToCheck)
         {
-            if (string.IsNullOrEmpty(fileExtensionToCheck))
-            {
-                return false;
-            }
+            return new[] { ".zip", ".msix", ".appx", ".msixbundle", ".7z", ".rar", ".tar" }
+                .Contains(fileExtensionToCheck, StringComparer.OrdinalIgnoreCase);
+        }
 
-            return fileExtensionToCheck.Equals(".zip", StringComparison.OrdinalIgnoreCase) ||
-                   fileExtensionToCheck.Equals(".msix", StringComparison.OrdinalIgnoreCase) ||
-                   fileExtensionToCheck.Equals(".msixbundle", StringComparison.OrdinalIgnoreCase);
+        public static bool IsBrowsableZipFile(string filePath, out string ext)
+        {
+            ext = new[] { ".zip", ".7z", ".rar", ".tar" } // Only ext we want to browse
```

Comment:
Can we add msix and appx to this list? They're basically self-extracting zip files.

Reply:
Not here, because then double clicking on them would browse the as archives instead of installing them. 
Above I believe I added msix and msixappxbundle to show the extract menu on them.